providers/cephfs.rb

```diff
@@ -29,6 +29,7 @@ def manage_mount(directory, subdir, use_fuse, action)
       dump 0
       pass 0
       action action
+      not_if "mount | grep \"^ceph-fuse on #{Regexp.escape(directory)}\"" if action == :mount
```

Comment:
Mounting fuse-based filesystem does not play well when the system reboots: the fstab entry is mounted again by the system, but the "mount" resource does not see that the volume is mounted and tried mounting again, which fails the chef-run. Interested in more pretty workarounds.